contracts/protocol/bases/OfferBase.sol

```diff
@@ -177,15 +173,16 @@ contract OfferBase is ProtocolBase, IBosonOfferEvents {
                 uint256 feeIndex = lookups.disputeResolverFeeTokenIndex[_disputeResolverId][_offer.exchangeToken];
                 require(feeIndex > 0, DR_UNSUPPORTED_FEE);
 
-                uint256 feeAmount = disputeResolverFees[feeIndex - 1].feeAmount;
+                // uint256 feeAmount = disputeResolverFees[feeIndex - 1].feeAmount;
```

Comment:
I believe that we should simply keep this in. Yes, it is a redundant operation, since `feeAmount` will be always zero (which is effectively the same as it is now). But if we comment this out and want that the change is actually applied to diamon, we need to upgrade both OfferHandler and OrchestrationHandler.

Then when we finally enable non-zero fee amount, we'd need to do the upgrade again (or at least do the cuts to point back to the current implementation).

Reply:
I put it back